roles/rsyslog/tasks/inputs/ovirt/main.yml

```diff
@@ -1,12 +1,28 @@
 ---
 # Deploy configuration files
-- name: Install/Update oVirt logging packages and generate configuration files in /etc/rsyslog.d
+- name: Install/Update oVirt input packages and generate configuration files in /etc/rsyslog.d
   vars:
     __rsyslog_packages: "{{ __rsyslog_ovirt_prereq_packages + __rsyslog_ovirt_packages }}"
     __rsyslog_rules: "{{ __rsyslog_ovirt_rules }}"
-  include_role:
-    name: "{{ role_path | dirname | dirname | dirname }}"
-    tasks_from: deploy.yml
+  include_tasks:
+    file: deploy.yml
+
+- name: Create oVirt input configuration files in /etc/rsyslog.d
+  vars:
+    __rsyslog_input_name: "{{ ovirt_item.name }}"
+    __rsyslog_input_type: "{{ ovirt_item.type }}"
+    __rsyslog_packages: []
+    __rsyslog_rules: [ { "name": "input-ovirt-{{ ovirt_item.name }}", "type": "input", "weight": "21",
+                         "sections": [ { "options": "{{ lookup('template', 'input_template.j2') }}
```

Comment:
do you need a `\n` at the end of the line here?

Reply:
The generated file looks like this. (The indentation of the following `if ( $.send_targets_only == "true" )` part looks odd, but the newline seems added.)
```
$ cat 21-input-ovirt-ovirt_input.conf
# Ansible managed

if
  ($syslogtag == "ovirt_input")
  then {
    call elasticsearch_output
}
if
  ($syslogtag == "ovirt_input")
  then {
    call elasticsearch_output_ops
}
 if ( $.send_targets_only == "true" ) then {
 call default_files
 }


```